### ISSUE TITLE
[Accessibility][Supporting the platform-Button]:"Primary create account" button is not adopting the high contrast white\black modes under Compound button section

### ISSUE THREAD

**Issue by dollykumari5** [OfficeDev/office-ui-fabric-react#9686]
Detailed description of this issue can be found here [Sharepoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/ES4u1n2IVNNPuoO2dU5gywIBdNy61RcmtbihpLEKSTfzyA?e=M7f2Dv)

Component:Button

**Comment by khmakoto** [comment]
Hi @dollykumari5, I can confirm this issue is happening. I'll make a PR shortly to fix the issue. Thanks!
